tests/foreman/api/test_errata.py

```diff
@@ -99,53 +325,31 @@ def test_positive_filter_by_cve(self):
 
         @Assert: Errata is filtered by CVE.
 
-        @caseautomation: notautomated
-
-        @CaseLevel: System
-        """
-
-    @stubbed()
-    @tier3
-    def test_positive_list_affected_hosts(self):
-        """View a list of affected content hosts for an erratum
-
-        @id: 27711bbd-b94f-4135-9b61-b004bd1cd365
-
-        @Setup: Errata synced on satellite server.
-
-        @Steps:
-
-        1. GET /katello/api/hosts
-
-        @Assert: List of affected content hosts for the given erratum is
-        retrieved.
-
-        @caseautomation: notautomated
-
         @CaseLevel: System
         """
+        repo_with_cves_id = enable_rhrepo_and_fetchid(
+            basearch=DEFAULT_ARCHITECTURE,
+            org_id=self.org.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhva6']['name'],
+            reposet=REPOSET['rhva6'],
+            releasever=DEFAULT_RELEASE_VERSION,
+        )
+        repo = entities.Repository(id=repo_with_cves_id)
+        self.assertEqual(repo.sync()['result'], 'success')
+        erratum_list = entities.Errata(repository=repo).search(query={
+            'order': 'cve DESC',
+            'per_page': 1000,
+        })
+        # Most of Errata don't have any CVEs. Removing empty CVEs from results
+        erratum_cves = [
+            errata.cves for errata in erratum_list if errata.cves
+        ]
+        # Verifying each errata have its CVEs sorted in DESC order
+        for errata_cves in erratum_cves:
+            cve_ids = [cve['cve_id'] for cve in errata_cves]
+            self.assertEqual(cve_ids, sorted(cve_ids, reverse=True))
 
-    @stubbed()
-    @tier3
-    def test_positive_filter_by_affected_hosts(self):
```

Comment:
There's no way to do that via API - neither some `affected_hosts` field is present in errata, nor some extra path exists. Instead, we can do the opposite thing - fetch applicable erratum for specific host. So i removed this stub and added `test_positive_get_applicable_for_host`